regen-network/regen-web
# fix:  render photo credit field only for images in EditFileForm

## Description

https://regennetwork.atlassian.net/browse/APP-235

---

### Author Checklist

I have...


### How to test

1. Create a new post
2. Add a new file:
   * If the file is an image the `Photo Credit ` field should be visible
   * If the file is not an image the `Photo Credit ` field should not be visible


### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
